script/migrations/utils/MigrationHelper.sol

```diff
@@ -103,63 +120,84 @@ contract MigrationHelper is Create2Helper, ArtifactsReader, UserOp {
         IEntryPoint(_getChainDeployment("EntryPoint")).handleOps(userOps, payable(vm.addr(_signerPk)));
     }
 
+    // TODO: should be extended to work with other initalize() that receive params
     function _initialize(address _proxy, uint256 _signerPk) internal {
-        address payable _from = payable(_getChainDeployment("KintoWallet-admin"));
-
         // fund _proxy in the paymaster
-        require(
-            ISponsorPaymaster(payable(_getChainDeployment("SponsorPaymaster"))).balances(_proxy) > 0,
-            "Need to fund proxy in paymaster"
-        );
+        if (ISponsorPaymaster(payable(_getChainDeployment("SponsorPaymaster"))).balances(_proxy) == 0) {
```

Comment:
This is needed only if the contract is not whitelisted by geth righ?

Reply:
good call